content/influxdb/v2.0/write-data/developer-tools/csv.md

```diff
@@ -16,6 +16,19 @@ related:
   - /influxdb/v2.0/reference/syntax/annotated-csv/
   - /influxdb/v2.0/reference/cli/influx/write/
 ---
+Write CSV data with the following methods:
+- [Flux](#flux)
+- [influx write command](#influx-write-command)
+
+### Flux
+
+Use the [csv.from()](/influxdb/v2.0/reference/flux/stdlib/csv/from/) and [to()](/influxdb/v2.0/reference/flux/stdlib/built-in/outputs/to/) Flux functions to write an annotated CSV to the bucket of your choice.
+ 
+{{< youtube wPKZ9i0DulQ >}}
+
+Try writing the [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data).
```

Comment:
- Provide the actual Flux example here.
- You should also mention the experimental `csv.from()` function which lets you write CSV from a URL.
- Instead of telling the reader to "try writing," say:

~~~
The example below writes [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data) to an example `noaa` bucket in an example organization:

```js
import "experimental/csv"

csv.from(url: "https://influx-testdata.s3.amazonaws.com/noaa.csv")
  |> to(bucket: "noaa", org: "example-org")
```
~~~